frontend/src/metabase/dashboard/components/DashboardInfoSidebar/components/InsightsUpsellTab.tsx

```diff
@@ -0,0 +1,35 @@
+import { Link } from "react-router";
+import { t } from "ttag";
+
+import { UpsellCard } from "metabase/admin/upsells/components";
+import { Box, Flex, Text } from "metabase/ui";
+
+const usageAnalyticsIllustrationSource = "app/assets/img/usage-analytics.png";
+
+export const InsightsUpsellTab = () => {
+  return (
+    <Flex justify="center">
+      <Box
+        component={UpsellCard}
+        variant="large"
+        title={t`See who’s doing what, when`}
+        buttonText={t`Try for free`}
+        buttonLink={"FIXME"}
+        campaign="FIXME"
+        source="FIXME"
```

Comment:
FIXWE

Reply:
After consulting Notion and some other upsells, I've added:
```
buttonLink="https://www.metabase.com/upgrade"
        campaign="usage-analytics"
        source={`${model}-insights-upsell-tab`}
```
I came up with that source, I don't know if that's right.